actions/authActions.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  setAuthToken: function(token) {
+    return { type: 'SET_AUTHTOKEN', token };
+  },
+
+  setCurrentUser: function(id) {
+    return { type: 'SET_CURRENT_USER', id };
+  },
+
+  clearAuthToken: function() {
+    return { type: 'CLEAR_AUTHTOKEN', token: null };
```

Comment:
It never has a hyphen anywhere else, it's normally AuthToken, but the convention here is to use only capital letters and underscores, hence "AUTHTOKEN". Also, this comment uses an underscore & the other uses a hyphen.